### PR TITLE
Test: jest config and first test

### PR DESCRIPTION
**Pull request by BCerki**
Some tests for #26

### CODE REVIEW

File: client/app/components/routes/operations/Operations.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { GridRowsProp } from "@mui/x-data-grid";
-import DataGrid from "@/components/datagrid/DataGrid";
-import { fetchAPI } from "@/utils/api";
+import DataGrid from "../../datagrid/DataGrid";
```
Comment (BCerki) [inline client/app/components/routes/operations/Operations.tsx:2]: @shon-button, my app couldn't find these imports until I changed them to the .. vs @. Have I got something set up wrong?

Comment (shon-button) [reply client/app/components/routes/operations/Operations.tsx:2]: I have no issues on this branch when using import for any of the tsconfig.json paths such as: import DataGrid from "@/components/datagrid/DataGrid";  Perhaps, it's a cache issue on your side needing a re-start of the dev server?

Comment (shon-button) [reply client/app/components/routes/operations/Operations.tsx:2]: For client/app/components/routes/operations/Operations.tsx imports:

**yarn dev** works with either:  "@/components/datagrid/DataGrid";  "@/app/components/datagrid/DataGrid"; "../../datagrid/DataGrid";
**yarn test** works with only: "@/app/components/datagrid/DataGrid";  "../../datagrid/DataGrid";

**yarn test** with import DataGrid from "@/components/datagrid/DataGrid"; errors with: Cannot find module '../../../../components/datagrid/DataGrid' from 'app/components/routes/operations/Operations.tsx'

Comment (shon-button) [reply client/app/components/routes/operations/Operations.tsx:2]: Suggest using only path alias @/* for imports to ensure proper jest test functionality.

e.g.
❌  "@/components/datagrid/DataGrid";
✔️  "@/app/components/datagrid/DataGrid";


❌  "@/utils/api";
✔️  "@/app/utils/api";

Comment (shon-button) [reply client/app/components/routes/operations/Operations.tsx:2]: Of interest?
[Next.js with React Testing Library, Jest, TypeScript](https://www.youtube.com/watch?v=AS79oJ3Fcf0)
[repo](https://github.com/gitdagray/next-testing-intro/tree/main)

Comment (shon-button) [reply client/app/components/routes/operations/Operations.tsx]: If agreement to use one path alias @/* for imports then the following files can be updated to reflect the single path alias:
- tsconfig.json\paths
- jest.config.mjs\ moduleNameMapper

**Review by shon-button — APPROVED** [review]

**Comment by shon-button** [comment]
From [MUI docs](https://mui.com/x/react-data-grid/virtualization/#column-virtualization): 
The virtualization can be disabled completely using the disableVirtualization prop. You may want to turn it off to be able to test the data grid with a headless browser, like jsdom.

Change made to client/app/components/datagrid/DataGrid.tsx
```
 <div style={{ height: "auto", width: "90%" }}>
      <MuiGrid rows={rows} columns={customColumns} disableVirtualization />
    </div>
```

**Comment by BCerki** [comment]
> disableVirtualization

Amazing, thank you!

**Pull request merged**